packages/context/src/interceptor.ts

```diff
@@ -197,15 +203,26 @@ export class InvocationContext extends Context {
 /**
  * The `BindingTemplate` function to configure a binding as a global interceptor
  * by tagging it with `ContextTags.INTERCEPTOR`
- * @param binding - Binding object
+ * @param group - Group for ordering the interceptor
  */
 export function asGlobalInterceptor(group?: string): BindingTemplate {
   return binding => {
-    binding.tag(ContextTags.GLOBAL_INTERCEPTOR);
+    binding
+      .tag(ContextTags.GLOBAL_INTERCEPTOR)
+      .tag({[ContextTags.NAMESPACE]: GLOBAL_INTERCEPTOR_NAMESPACE});
```

Comment:
I am confused. Why do we need both `GLOBAL_INTERCEPTOR` tag and `GLOBAL_INTERCEPTOR_NAMESPACE`? I mean why is it not enough to use the tag only?

Is it a good idea to use the same namespace for all interceptors, regardless of which component/extension is contributing them? Wouldn't it better to use namespaces similarly to how Java treat packages, i.e. interceptors contributed by `@loopback/authorization` package should use `@loopback/authorization` namespace.

Let's discuss.

Reply:
Only `GLOBAL_INTERCEPTOR` is required. `GLOBAL_INTERCEPTOR_NAMESPACE` is to make the binding key more readable.

Reply:
Please add a code comment to capture this information.

Also, what is your opinion about using the same namespace for all interceptors vs. using a namespace based on the component/extension contributing the interceptor?

Reply:
Personally, I prefer to use `globalInterceptors.*`.